comparison.html

```diff
@@ -0,0 +1,229 @@
+        // lcEnd: "2009-08-05",
+
+        // editors, add as many as you like
+        // only "name" is required
+        editors:  [
+          { name: "David Chadwick", url: "https://www.linkedin.com/in/david-chadwick-36816395/",
+            company: "University of Kent", companyURL: "https://www.kent.ac.uk/"},
+          { name: "Manu Sporny", url: "http://digitalbazaar.com/",
+            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/" }
+        ],
+
+        // authors, add as many as you like.
+        // This is optional, uncomment if you have authors as well as editors.
+        // only "name" is required. Same format as editors.
+        authors:
+        [
+          { name: "David Chadwick", url: "https://www.linkedin.com/in/david-chadwick-36816395/",
+            company: "University of Kent", companyURL: "https://www.kent.ac.uk/"}
+        ],
+
+        // name of the WG
+        wg:           "Verifiable Claims Working Group",
+
+        // URI of the public WG page
+        wgURI:        "https://www.w3.org/2017/vc/",
+
+        // name (with the @w3c.org) of the public mailing to which comments are due
+        wgPublicList: "public-vc-comments",
+
+        // URI of the patent status for this WG, for Rec-track documents
+        // !!!! IMPORTANT !!!!
+        // This is important for Rec-track documents, do not copy a patent URI from a random
+        // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
+        // Team Contact.
+        wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/98922/status",
+        maxTocLevel: 4,
+        inlineCSS: true
+      };
+    </script>
+    <style>
+pre .highlight {
+  font-weight: bold;
+  color: green;
+}
+pre .subject {
+  font-weight: bold;
+  color: RoyalBlue;
+}
+pre .property {
+  font-weight: bold;
+  color: DarkGoldenrod;
+}
+pre .comment {
+  font-weight: bold;
+  color: Gray;
+}
+</style>
+  </head>
+  <body>
+    <section id='abstract'>
+      <p>
+This document is a comparison of Verifiable Claims, SAML, and OpenID Connect.
+      </p>
+    </section>
+
+    <section id='sotd'>
+      <p>
+Comments regarding this document are welcome. Please file issues
+directly on <a href="https://github.com/w3c/vc-data-model/issues/">GitHub</a>,
+or send them to
+<a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+(<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
+<a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).
+      </p>
+    </section>
+
+    <section>
+      <h1>Comparison of Verificable Credentials with SAML and OIDC</h1>
+
+      <p>
+Existing federated identity management (FIM) systems suffer from a
+number of problems, in particular with regard to their trust model,
+economic models and the privacy of their users.
+      </p>
+
+      <p>
+The FIM trust model is wrong as it requires the Identity Provider (IdP)
```

Comment:
Let's strike "is wrong as it". It's a subjective value judgment that detracts from the point.

Reply:
Accepted

Reply:
Fixed.